demo-circuit/src/naive_threshold_sig_w_key_rotation/tests/verify.rs

```diff
@@ -412,12 +457,50 @@ fn malicious_master_key_rotation() {
     assert!(circuit_res.is_ok());
     let circuit = circuit_res.unwrap();
 
-    debug_naive_threshold_circuit(&circuit, true, Some("check key changes/check updated signing key should be signed old master key 0/conditional verify signature/conditional_equals"));
+    debug_naive_threshold_circuit(&circuit, true, Some("enforce new root equals the one in curr cert/conditional_equals"));
+}
+
+#[serial]
+#[test]
+fn test_replay_attack() {
```

Comment:
Can we add some comment about the motivation of this test ?

Reply:
Ok